tar/Dockerfile

```
@@ -0,0 +1,6 @@
+FROM launcher.gcr.io/google/ubuntu16_04
```

What about using the Alpine base image instead?  It has minimal packages installed, so loads much faster and has a far smaller attack surface.  It's also cached on Cloud Build workers, and reduces build time from 35 seconds to 14 seconds.
```suggestion
FROM alpine
```